inputfiles/addedTypes.jsonc

```diff
@@ -449,6 +449,38 @@
     },
     "interfaces": {
         "interface": {
+            // ImportMeta is not a true DOM interface, but we are forced to declare it as one in order to allow method definitions.
```

Comment:
Mentioning @types/node would be helpful because otherwise we wouldn't need a "method".